src/_cffi_src/openssl/engine.py

```diff
@@ -9,8 +9,6 @@
 """
 
 TYPES = """
-static const long Cryptography_HAS_ENGINE_CRYPTODEV;
```

Comment:
Should we leave this constant but set to `0`? Then there's no plausible backwards compat concern :-)